### PR TITLE
geopard: init at 1.4.0

### LINKED ISSUE #259429: Package request: Geopard
**Project description**
 Colorful, adaptive gemini browser 

**Metadata**

* homepage URL: https://ranfdev.com/projects/geopard/
* source URL: https://github.com/ranfdev/Geopard
* license: gpl3
* platforms: linux

### PR DESCRIPTION
**Pull request by jfvillablanca**
## Description of changes
Resolves #259429 

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by jfvillablanca** [comment]
Result of `nixpkgs-review pr 260205` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>geopard</li>
  </ul>
</details>

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -8518,6 +8518,8 @@ with pkgs;
 
   geonkick = callPackage ../applications/audio/geonkick { };
 
+  geopard = callPackage ../applications/networking/browsers/geopard {};
```
Comment (nagy) [inline pkgs/top-level/all-packages.nix:8521]: can you use the new by-name structure https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#readme ?

**Review by nagy — CHANGES_REQUESTED** [review]
also, can you drop the v prefix from your commit message?

File: pkgs/top-level/all-packages.nix (hunk 1 above)
Comment (jfvillablanca) [reply pkgs/top-level/all-packages.nix:8521]: thanks for the heads up

Comment (jfvillablanca) [reply pkgs/top-level/all-packages.nix:8521]: I don't understand the error in the CI, can you help me? Thanks

Comment (nagy) [reply pkgs/top-level/all-packages.nix:8521]: you still have two commits. maybe removing the older one helps.

**Review by nagy — APPROVED** [review]

File: pkgs/by-name/ge/geopard/package.nix [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    appstream-glib
+    blueprint-compiler
+    cargo
+    rustc
+    rustPlatform.cargoSetupHook
+  ];
+
+  buildInputs = [
+    meson
+    ninja
```
Comment (wegank) [inline pkgs/by-name/ge/geopard/package.nix:58]: I rarely see `meson` or `ninja` in `buildInputs`. Could you check if they are needed here?

**Pull request merged**